plugins/palo_alto_cortex_xdr/icon_palo_alto_cortex_xdr/util/api.py

```diff
@@ -182,10 +198,11 @@ def get_alerts(
         )
 
     def get_incidents(
-        self, from_time: int, to_time: int, time_sort_field: str = "creation_time", filters: List = None
+        self, from_time: int = None, to_time: int = None, time_sort_field: str = None, filters: List = None
```

Comment:
Why have we stopped sorting incidents by creation_time?

Reply:
I have made changes to now sort from `creation_time`